### PR TITLE
Optionally return the anwser content in semantic search

### PR DESCRIPTION
**Pull request by mruwnik**

### CODE REVIEW

File: src/stampy_nlp/search.py [hunk 1]
```diff
@@ -27,7 +28,17 @@ def format_matches(results):
     return results_list
 
 
-def semantic_search(query: str, top_k: int = DEFAULT_TOPK, showLive: bool = True, status=[], namespace: str = 'faq-titles'):
+def add_doc_content(results):
+    def update(result):
+        result['content'] = get_contents(result['pageid'])
+        return result
+
+    with concurrent.futures.ThreadPoolExecutor(max_workers=3) as executor:
```
Comment (mruwnik) [inline src/stampy_nlp/search.py:36]: it turns out that Coda doesn't have a nice way to filter rows :/ You can either fetch them one by one, or loop over the whole table. Both would involve multiple calls, this way may hopefully be faster?

**Review by ccstan99 — APPROVED** [review]
Awesome!! Would you please let @joepio know that he can use the `getContent` parameter when you've got it publicly deployed?

**Comment by joepio** [comment]
> Awesome!! Would you please let @joepio know that he can use the `getContent` parameter when you've got it publicly deployed?

I now know!

**Comment by mruwnik** [comment]
Yup. Deployed and checked, both CORS and the getContent param should work for nlp.stampy.ai now

**Pull request merged**